### PR TITLE
Dev dout get state

### PR DESCRIPTION
**Pull request by sjpark608**
get_state() method implemented to retrieve the current state of the specified DOUT pin

### CODE REVIEW

**Review by farzadpanahi — APPROVED** [review]
let's just do the refactoring for gpio pins as we discussed to separate enums per module.

File: src/test_edgepi/integration_tests/test_digital_out/test_digital_out.py [hunk 1]
```diff
@@ -34,3 +36,39 @@ def test_output_low(pin_name):
...
+    (GpioPins.DOUT4),
+    (GpioPins.DOUT5),
+    (GpioPins.DOUT6),
+    (GpioPins.DOUT7),
+    (GpioPins.DOUT8),
+])
+def test_direction_in(pin_name):
+    dout=EdgePiDigitalOutput()
+    dout.digital_output_direction(pin_name, True)
+    time.sleep(1)
```
Comment (naomiven) [inline src/test_edgepi/integration_tests/test_digital_out/test_digital_out.py:55]: what is this sleep for?

**Review by naomiven — APPROVED** [review]
looks good

**Review by jarell-santella — APPROVED** [review]

File: src/test_edgepi/integration_tests/test_digital_out/test_digital_out.py (hunk 1 above)
Comment (sjpark608) [reply src/test_edgepi/integration_tests/test_digital_out/test_digital_out.py:55]: it was there to check the LED response when DOUT state changes

**Pull request merged**